tests/lightfm/test_lenskit_lightfm.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import pickle
+
+import numpy as np
+import pandas as pd
+from pytest import approx, mark
+import sys 
+import os 
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+
+from lenskit.data import Dataset, ItemList, from_interactions_df
+from lenskit.metrics import quick_measure_model
+from lenskit.testing import BasicComponentTests, ScorerTests
+from lenskit_lightfm import LightFMScorer
```

Comment:
changing this to `from lenskit.lightfm` should fix the immediate test failure problem.